src/main/scala/mimir/models/ModelRegistry.scala

```diff
@@ -98,8 +98,8 @@ object ModelRegistry
    * to add new models here.
    */
   val imputations = Map[String,ImputationConstructor](
-    //"WEKA" -> (WekaModel.train _)
-      "SPARK" -> (SparkClassifierModel.train _)
+    ("SERIES" -> (SeriesMissingValueModel.train _)),
+    ("WEKA" -> (WekaModel.train _))
```

Comment:
Broken Merge.  Should be using SPARK and SERIES.  Weka is deprecated (correct me if I'm wrong @mrb24).